xtype/type.go

```diff
@@ -108,14 +124,14 @@ func (t Type) findAllFields(path []string, name string, ignoreCase bool) (*Struc
 	}
 
 	for y := 0; y < t.StructType.NumFields(); y++ {
-		if exact := handle(t.StructType.Field(y)); exact != nil {
+		if exact := handle(t.StructType, y); exact != nil {
 			return exact, matches
 		}
 	}
 
 	if t.Named {
 		for y := 0; y < t.NamedType.NumMethods(); y++ {
-			if exact := handle(t.NamedType.Method(y)); exact != nil {
+			if exact := handle(t.NamedType, y); exact != nil {
```

Comment:
I think the handle method could be simpler, if we extract the tag here directly:
```
handle(t.StructType.Field(y), t.StructType.Tag(y));
```
for the named stuff below, we can pass empty string

```
handle(t.NamedType.Method(y), "");
```